src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -243,32 +243,38 @@
-                                                     :discard-dir          (.getAbsolutePath discard-dir)
-                                                     :mq-addr              mq-addr
-                                                     :mq-dest              mq-endpoint
-                                                     :mq-threads           (:threads command-processing)
-                                                     :catalog-hash-debug-dir (:catalog-hash-debug-dir global)}]
+  (let [{:keys [global   jetty
+                database read-database
+                puppetdb command-processing]}   (conf/process-config! config)
+        {:keys [product-name update-server
+                vardir catalog-hash-debug-dir]} global
+        {:keys [gc-interval    node-ttl
+                node-purge-ttl report-ttl]}     database
+        {:keys [dlo-compression-threshold
+                 max-frame-size threads]}       command-processing
+        {:keys [certificate-whitelist
+                disable-update-checking]}       puppetdb
+        url-prefix                              (get-route service)
+        write-db                                (pl-jdbc/pooled-datasource database)
+        read-db                                 (pl-jdbc/pooled-datasource (assoc read-database :read-only? true))
+        mq-dir                                  (str (file vardir "mq"))
+        discard-dir                             (file mq-dir "discarded")
+        mq-connection-str                       (add-max-framesize max-frame-size mq-addr)
+        authorizer                              (when certificate-whitelist
+                                                  (build-whitelist-authorizer certificate-whitelist))
+        globals                                 {:scf-read-db            read-db
+                                                 :scf-write-db           write-db
+                                                 :authorizer             authorizer
+                                                 :command-mq             {:connection-string mq-connection-str
+                                                                          :endpoint          mq-endpoint}
+                                                 :update-server          update-server
+                                                 :product-name           product-name
+                                                 :url-prefix             url-prefix
+                                                 :discard-dir            (.getAbsolutePath discard-dir)
+                                                 :mq-addr                mq-addr
+                                                 :mq-dest                mq-endpoint
+                                                 :mq-threads             threads
+                                                 :catalog-hash-debug-dir catalog-hash-debug-dir}]
 
```

Comment:
I don't know what the consensus is (if there is one), but I'd likely vote for removing the column formatting when we're changing code.